e2e/app/page/home-page.ts

```diff
@@ -46,8 +46,17 @@ export default class HomePage extends AuthenticatedPage {
           logger.info('Home page is empty without workspace card');
         }
       });
+      try {
+        // Look for either Recent Resources table or the Getting Started text.
+        await Promise.race([
+          this.page.waitForXPath('//*[@data-test-id="recent-resources-table"]', { visible: true, timeout: 2000 }),
+          this.page.waitForXPath('//*[@data-test-id="getting-started"]', { visible: true, timeout: 2000 })
+        ]);
+      } catch (err) {
+        // ignore error
+      }
```

Comment:
Need the additional checks to avoid `isLoaded` exiting early before Home page is ready.